### DISCUSSION TITLE
How can I check the Vercel Edge Funcitons logs?

### DISCUSSION THREAD

**Discussion by ogawa0071** [vercel/community#158] [Idea]
The log of Vercel Edge Functions could not be confirmed in "Function Logs" of Vercel console and Datadog linked with Log Drain.

How can I check it?

**Comment by Pieparker** [comment]
Hey there! Is this still an issue for you? If not, how did you overcome this? If it is, can you share more about the errors you're seeing, what you're trying, and a slug or deployment ID?

**Reply by ogawa0071** [reply]
I tried the following, but it did not solve the problem.

- Open the log in Vercel's console and have a look.
- Output to Datadog using Log Drain integration

**Reply by leerob** [reply]
Thanks for letting us know - we will look into this for you and report back!

**Reply by Pieparker** [reply]
Team are still looking in to this! We'll loop back once we know more.

internal ref: https://github.com/vercel/customer-issues/issues/134

**Comment by Pieparker** [comment]
Hey there! I just wanted to update you to share that we're looking to expand our log capabilities to cover more log-able items. Please do keep an eye out in our changelog for when this is released! Our team will endeavour to reply to this post once it launches too.
